examples/dash/dash-connect-pgsql/README.md

```diff
@@ -0,0 +1,19 @@
+# Dash App Connected to PostgreSQL Database
+
+Interactive Dash Application, connected to PostgreSQL database.
+
+![](app.png)
+
+To use the app, store the below information locally into your `.env` for `upload.py`. These variables can be found on the `Parameters only` section under `connection details` from your PostgreSQL dashboard.
```

Comment:
Please mention dashboard of which application?

Reply:
[fixed](https://github.com/ploomber/doc/pull/259/commits/b79148166cd7baddea39651ff6eabe157ca80edf#diff-947610ca10aacde7e2a54afa9c3fbd37fea9055230c4a6222dc39decedc110afR8)